scripts/snowpack.base.config.js

```diff
@@ -78,11 +78,13 @@ module.exports = {
   optimize: isProd()
     ? {
         souecemap: 'inline',
-        // bundle: true, // TODO: turn this on after https://github.com/snowpackjs/snowpack/issues/3403 is resolved
+        // TODO: turn this on after https://github.com/snowpackjs/snowpack/issues/3403 is resolved
```

Comment:
seems it has been turned on?

Reply:
it's resolved in @yqrashawn/snowpack

Reply:
But still wait the snowpack issue resolving. it's only a temporary workaround